tests/Doctrine/Tests/ORM/Functional/QueryTest.php

```diff
@@ -317,38 +317,30 @@ public function testIterateResultClearEveryCycle() : void
         $this->em->flush();
     }
 
-    /**
-     * @expectedException \Doctrine\ORM\Query\QueryException
-     */
     public function testIterateResultFetchJoinedCollectionThrowsException() : void
     {
+        $this->expectException('Doctrine\ORM\Query\QueryException');
```

Comment:
```suggestion
        $this->expectException(\Doctrine\ORM\Query\QueryException::class);
```

plus `use`.

This change should be done with the other string class name ocurances too. I'm too lazy to add a review comment to the other ones. :-)